backend/doc/setting.md

```diff
@@ -474,18 +492,43 @@ If there is an update of tracim, use this one to migrate index (experimental, pr
 
 Your data are correctly indexed now, you can go to tracim ui and use search mecanism.
 
-# Collaborative Edition online #
+# Collaborative Edition online (tracim v2.4+) #
 
 ## Collaborative edition server ##
-At the moment tracim is only tested with [Collabora CODE](https://www.collaboraoffice.com/code/), but Tracim is theoretically ready for any software using the protocol `WOPI`.
 
-All the informations to set up a `Collabora CODE` server can be found on the [official documentation](https://www.collaboraoffice.com/code/docker/)
+In tracim 2.4, Collaborative Edition online does support Collabora/libreofficeOnline.
+
+It is tested with Collabora Online (professional version of Collabora) and with [Collabora CODE](https://www.collaboraoffice.com/code/).
+We do not support yet other collaborative edition online service but we do support WOPI protocol making support for WOPI compatible software easy.
+
+**To set up a `Collabora CODE` server using docker for testing purpose ([image](https://hub.docker.com/r/collabora/code)):**
+
+note: you should replace <DOT_ESCAPED_DOMAIN_OF_TRACIM_API> with real value like `domain=tracim\\.mysuperdomain\\.com`):
+
+
+`sudo docker run -d -t -p 9980:9980 -e "domain=<DOT_ESCAPED_DOMAIN_OF_TRACIM_API>" -e "SLEEPFORDEBUGGER=0" -e "extra_params=--o:ssl.enable=false" --cap-add MKNOD --restart always collabora/code:4.0.5.2`
+
+:warning: Tracim is tested with the version 4.0.5.2, you can use the latest version at your own risk
 
-To set up a `Collabora CODE` server using docker for testing purpose:
 
-`sudo docker run -d -t -p PORT:PORT -e "domain=IP_OF_TRACIM_API" -e "username=admin" -e "password=S3cRet" -e "SLEEPFORDEBUGGER=0" -e "extra_params=--o:ssl.enable=false" --cap-add MKNOD --restart always collabora/code:4.0.4-3`
+**To set up a `LibreofficeOnline` server(rolling release, unstable :warning:) using docker ([image](https://hub.docker.com/r/libreoffice/online)):**
+
+
+`sudo docker run -d -t -p 9980:9980 -e "domain=<DOT_ESCAPED_DOMAIN_OF_TRACIM_API>" -e "SLEEPFORDEBUGGER=0" -e "extra_params=--o:ssl.enable=false" --cap-add MKNOD --restart always libreoffice/online:master`
+
+
+:information_source: All the information to set up a `Collabora CODE/ LibreofficelOnline` server can be found on the [official documentation](https://www.collaboraoffice.com/code/docker/)
+
+:warning: Be really careful about configuring domain parameter, as [official documentation](https://www.collaboraoffice.com/code/docker/) said, value should be dot escaped, ex: `domain=.*\\.mysuperdomain\\.com`
+
+:information_source: you can add username/password parameter to set collabora admin console too :
```

Comment:
I think it's necessary to add link where I can find more information like url to use this admin console.